src/lib/include/roc/address.h

```diff
@@ -85,6 +85,19 @@ typedef struct roc_address {
 ROC_API int
 roc_address_init(roc_address* address, roc_family family, const char* ip, int port);
 
+/** Set multicast interface address.
+ *
+ * @b Parameters
+ *  - @p address should point to a properly initialized address struct
+ *  - @p miface should point to a zero-terminated string with a valid IPv4 or
+ *       IPv6 multicast address
+ *
+ * @b Returns
+ *  - returns zero if no error occurred
+ *  - returns a negative value if the arguments are invalid
+ */
+ROC_API int roc_address_set_miface(roc_address* address, const char* miface);
```

Comment:
Let's be explicit and rename it to `roc_address_set_multicast_interface`.

Reply:
Done